gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/BaseDistributionExtensionTest.groovy

```diff
@@ -35,4 +35,55 @@ class BaseDistributionExtensionTest extends Specification {
         def ex = thrown IllegalArgumentException
         ex.message == "Invalid product type 'foobar' specified; supported types: [service.v1, asset.v1]."
     }
+
+    def "productDependencies from invalid maven coordinate"() {
+        when:
+        def ext = new BaseDistributionExtension(null)
+        ext.productDependency("group:name")
```

Comment:
with this, you would call, for instance:

    distribution {
    ...
        productDependency recommend('group:name')
    }